includes/class-wc-payments.php

```diff
@@ -400,7 +400,7 @@ public static function init() {
 		include_once __DIR__ . '/platform-checkout/class-platform-checkout-utilities.php';
 		include_once __DIR__ . '/platform-checkout/class-platform-checkout-order-status-sync.php';
 		include_once __DIR__ . '/class-wc-payment-token-wcpay-link.php';
-		include_once __DIR__ . '/core/services/class-wc-payments-customer-service-api.php';
+		include_once __DIR__ . '/core/service/class-wc-payments-customer-service-api.php';
```

Comment:
No, there will definitely be more than one service in there at some point. :)

This is mostly done for consistency. The server requests already use singulars, which was done because it's that way on the server as well. Context: https://github.com/Automattic/woocommerce-payments-server/pull/2606#pullrequestreview-1156345179

Reply:
Thanks for providing context